action.yml

```diff
@@ -19,10 +19,28 @@ inputs:
     description: 'Toggles maintenance mode during the deployment. Default value "true" enables the maintenance mode.'
     required: false
     default: 'true'
+  api_key_statuscake:
+    description: 'API key for StatusCake'
+    required: false
+  test_id_statuscake:
+    description: 'Test ID in StatusCake'
+    required: false
 
 runs:
   using: "composite"
   steps:
+    - name: 'Disable StatusCake monitoring (if applicable)'
+      shell: bash
+      run: |
+        if [ -n "${{ inputs.api_key_statuscake }}" ] && [ -n "${{ inputs.test_id_statuscake }}" ]; then
```

Comment:
Have you tried using `if` - See https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/using-conditions-to-control-job-execution